### ISSUE TITLE
Export error in Blender 4.2.1 LTS

### ISSUE THREAD

**Issue by hilburger** [KhronosGroup/glTF-Blender-IO#2336]
**Describe the bug**
I get the followinf error trying to export selected objects from a Blender scene (4.2.1 LTS): 

```
Python: Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\__init__.py", line 1288, in execute
    res = gltf2_blender_export.save(context, export_settings)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_export.py", line 37, in save
    json, buffer = __export(export_settings)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_export.py", line 55, in __export
    __gather_gltf(exporter, export_settings)
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_export.py", line 202, in __gather_gltf
    active_scene_idx, scenes, animations = gltf2_blender_gather.gather_gltf2(export_settings)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_gather.py", line 31, in gather_gltf2
    scenes.append(__gather_scene(blender_scene, export_settings))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_gather_cache.py", line 37, in wrapper_cached
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_gather.py", line 94, in __gather_scene
    node = gltf2_blender_gather_nodes.gather_node(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_gather_nodes.py", line 42, in gather_node
    mesh = __gather_mesh(vnode, blender_object, export_settings)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\addons_core\io_scene_gltf2\blender\exp\gltf2_blender_gather_nodes.py", line 301, in __gather_mesh
    blender_mesh[prop] = blender_object.data[prop]
    ~~~~~~~~~~~~^^^^^^
TypeError: Cannot assign a 'IDPropertyGroup' value to the existing 'hops' Group IDProperty
```

**To Reproduce**
1. Select objects via collection in Blender.
2. Open export gltf dialogue
3. Setting glb options (see screenshot below)
4. Immediately after clicking on export the error message is shown. 

![image](https://github.com/user-attachments/assets/65dff2d9-5cef-49aa-ae26-77a2e163fde9)

**Expected behavior**
Export should be saved to the selected folder.

**.blend file/ .gltf (mandatory)**
I can't provide the file here because it's 400MB large and limit here is 25 ;)
Shall I upload it somewhere and link it?

**Version**
 - OS: Windows 10 + 11
 - Blender Version 4.2.1 LTS

**Additional context**
The error appears exactly identical on two different machines.

**Comment by julienduroure** [comment]
Hello,
Thanks for the reporting.
This should be solved on 4.3 alpha, by a658dbc287f6e3a049ec31d05d09cf56fccdaace
This commit is in proposed list of backport to 4.2.x

**Comment by hilburger** [comment]
> Hello, Thanks for the reporting. This should be solved on 4.3 alpha, by [a658dbc](https://github.com/KhronosGroup/glTF-Blender-IO/commit/a658dbc287f6e3a049ec31d05d09cf56fccdaace) This commit is in proposed list of backport to 4.2.x

Sound awesome! 
So either I wait until the next update for 4.2.x or I use 4.3,?

**Comment by julienduroure** [comment]
> So either I wait until the next update for 4.2.x or I use 4.3,?

Yes!
